docs/DeveloperGuide.md

```diff
@@ -2,12 +2,153 @@
 
 ## Acknowledgements
 
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the
+original source as well}
+
+[Original Source](https://github.com/nus-cs2113-AY2425S1/tp)
 
 ## Design & implementation
 
 {Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
 
+## Overall Class Diagram
+![overallClass.jpg](./Diagrams/overallClass.jpg)
```

Comment:
I think since you are already omitting other classes for conciseness, i dont think its necessary and easier to read to include simple getter functions in the classes that you show here.

Reply:
especially since this is to show the overall architecture this can be quite trivial